appgate/secrets.py

```diff
@@ -149,16 +229,19 @@ def get_appgate_secret(
         return AppgateSecretSimple(value, secrets_cipher)
     elif AppgateSecretSimple.isinstance(value):
         return AppgateSecretPlainText(value)
+    elif AppgateVaultSecret.isinstance(value):
```

Comment:
nice! you just extend vault as one the mechanism we already support for secrets

Reply:
Exactly! It worked out quite nicely so I followed a similar pattern for the files loader too.